Adds command `purview threatassessment list`, Closes #4426 (#4517)
pnp/cli-microsoft365: docs/docs/cmd/purview/threatassessment/threatassessment-list.md

Reviewer: We've recently upgraded our SSG to Docusaurus. This update has impacted our documentation syntaxes and will require some adjustments on your end. However, there's no need to worry, as Milan has done an amazing job creating a detailed page that explains all the new syntaxes to be used in the `.mdx` files. If you have any questions, feel free to reach out to us here or on Discord. You can find more information at this link: [All you need to know about Docusaurus](https://github.com/pnp/cli-microsoft365/wiki/All-you-have-to-know-about-Docusaurus).

# purview threatassessment list

> Get a list of threat assessment

Reviewer 2: ```suggestion
Get a list of threat assessments
```

## Usage

```sh
m365 purview threatassessment list [options]
```

## Options

`-t, --type [type]`
: The type of threat assessment to retrieve. Supports `mail`, `file`, `emailFile` and `url`.

--8<-- "docs/cmd/_global.md"

## Examples

> Get a list of threat assessment

Reviewer 2: ```suggestion
Get a list of threat assessments
```

```sh
m365 purview threatassessment list
```

> Get a list of threat assessment by type

Reviewer 2: ```suggestion
Get a list of threat assessments of type _mail_
```
